data/heliarch intro.txt

```diff
@@ -0,0 +1,1568 @@
+			`	"Lead the way, would you? The warehouses, I wish to inspect, but your assistance I could use."`
+			choice
+				`	"Of course, I'll get my rain coat."`
+					goto wetbeetle
+				`	"I'd rather not go out there with that rain."`
+					goto lonebeetle
+			label wetbeetle
+			`	You walk in a straight line for about an hour, many times without any building providing cover from the rain. Kirrie takes you to the very edge of the town; you would be able to see the hills and mountains in the distance, were it not for the colossal rain clouds and the nonstop precipitation blocking your view. One or two warehouses have a bit of rust on their doors, and rarely you spot one with a damaged roof, some rain creeping inside. You can also make out the figures of small trees and bushes though, not too far out.`
+			`	"Now, Captain, to start looking for something suspicious, where would you?" He asks, showing you some sort of identification card, and explaining that it can open the doors here, but you really can't make out any differences between any of the many warehouses here. One or two have a bit of rust on the doors, and some of them appear to have some rain dripping it's way inside, but aside from small details you can't make out any striking characteristic of the Arachi warehouses.`
+			choice
+				`	(Check the warehouses with the rusty doors.)`
+					goto rustybeetle
+				`	(Check the warehouses with infiltration.)`
+					goto reactor
+				`	(Go further out and check the trees and bushes.)`
+					goto trees
+			label rustybeetle
+			`	Kirrie uses his card to open the door, and continues to explain that it is an access card granted to Heliarch agents, in case they need to do something like this, or just identify themselves as one.`
+			`	Sadly the warehouse turns out to not contain anything significant. It's a storage for what Kirrie says are toy parts, just mountains of plastic stacked together.`
+			`	You check a few more like this, but none turn out to have anything. The rain is starting to get heavier.`
+			choice
+				`	(Check the warehouses with infiltration now.)`
+					goto reactor
+				`	(Go further out and check the trees and bushes.)`
+					goto trees2
+				`	(Leave the rest of this to Kirrie.)`
+					goto whiteflagbeetle
+			label whiteflagbeetle
+			`	You apologize and tell Kirrie that you don't know what to do next, but while he does agree to do it himself, he never stops asking for your opinion on his choices.`
+			`	You spend a few more hours checking the warehouses, until the rain and the wind get too dangerous for you to be moving from warehouse to warehouse, so you take shelter in one of them until the weather settles. Once it does, near nightfall, he gets notified by Iplora that she and Tugroob are already back at the ship, so you two decide to head back too.`
+			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
```

Comment:
`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. Exhausted, they get back in their bunks, asking you to depart the planet.`